CMakeLists.txt

```diff
@@ -15,16 +15,17 @@ option(BUILD_FORTRAN "Build the Fortran Mex libraries" OFF)
 list(APPEND CMAKE_MODULE_PATH
     "${CMAKE_SOURCE_DIR}/cmake"
     "${CMAKE_SOURCE_DIR}/cmake/external"
+    "${CMAKE_SOURCE_DIR}/cmake/shared"
 )
 
 # Look for packages early so we can exit if they're not found
 include(herbert_FindMPI)
```

Comment:
Is there any reason to leave `herbert_FindMPI` unchanged? The only `herbert` specific piece is `herbert_root` in a file path; that could become an argument to make it reusable